README.md

```diff
@@ -4,6 +4,14 @@ NYAN TAP output ~~inspired by~~ blatantly copied from Mocha's [nyan reporter](ht
 
 ![screen shot](http://i.imgur.com/T1eH156.png)
 
+**ANSI colors**
```

Comment:
this is no longer up to date, we should probably replace it with a reference to the supported flags in `supports-color`